eng/CodeAnalysis.src.globalconfig

```diff
@@ -483,6 +483,9 @@ dotnet_diagnostic.CA1863.severity = suggestion
 # CA1864: Prefer the 'IDictionary.TryAdd(TKey, TValue)' method
 dotnet_diagnostic.CA1864.severity = warning
 
+# CA1868: Unnecessary call to 'Contains' for sets
+dotnet_diagnostic.CA1868.severity = warning
```

Comment:
Can you add a corresponding entry to the .test.globalconfig file?  That one would be at level none.